lib/index.js

```diff
@@ -45,7 +43,15 @@ Drip.prototype.identify = function(identify, fn){
     .type('json')
     .send({ subscribers: [subscriber] })
     .end(function(err, res){
-      if (err) return fn(err);
+      if (err) {
```

Comment:
not sure if we need to do this grunt work, the err should capture the request and whatever was returned.